gsa/radix.go

```diff
@@ -1,20 +1,92 @@
 package gsa
 
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
 // Sort the string x using a count sort
 func CountSort(x string) string {
-	return ""
+	var sb strings.Builder
+	counts := make(map[byte]int)
```

Comment:
Here, you are assuming that your string can be reduced to its bytes. It is a fair assumption for ANSI text (and for our DNA strings in this class). It can go wrong because UTF-8 doesn't represent each unicode code point as a single byte, so the byte order isn't necessarily the alphabet order, but it *is* an assumption we generally make in this class, where we only want to handle DNA strings.